### PR TITLE
Eol request

### PR DESCRIPTION
**Pull request by alarku07**
* move speciescontroller to main module
* optimize eol request
* fix classification css

### CODE REVIEW

**Review by timo-piirmann — APPROVED** [review]

File: grails-app/views/species/tabs/_overview.gsp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section class="tab-pane active" id="tab-overview" role="tabpanel">
     <div class="row taxon-row">
-        <div class="col-md-5">
+        <div class="col-12 col-md-5">
```
Comment (raivop) [inline grails-app/views/species/tabs/_overview.gsp:3]: How does that wrok together?

File: grails-app/assets/javascripts/species.show.js [hunk 2]
```diff
@@ -256,6 +256,7 @@ function loadExternalSources() {
                     } else {
                         $description.find('.content').html(dataObject.description);
                     }
+                    $description.find('img').addClass('img-responsive');
```
Comment (raivop) [inline grails-app/assets/javascripts/species.show.js:259]: That's a very peculiar way of adding style.

Comment (alarku07) [reply grails-app/assets/javascripts/species.show.js:259]: i'd think that i'd do a better job if the content weren't pre-rendered

File: grails-app/views/species/tabs/_overview.gsp (hunk 1 above)
Comment (alarku07) [reply grails-app/views/species/tabs/_overview.gsp:3]: previously the map and list looked retarded

**Pull request merged**